recipe/meta.yaml

```diff
@@ -1,65 +1,69 @@
 {% set name = "spacy" %}
-{% set version = "3.5.3" %}
+{% set version = "3.7.2" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 35971d6721576538d6c423c66a09ce00bf66e10e40726a57b7a81993180c248c
+  sha256: cedf4927bf0d3fec773a6ce48d5d2c91bdb02fed3c7d5ec07bdb873f1126f1a0
+  patches:
+    - patches/0001-skip-beam-density-test.patch
 
 build:
   number: 0
   # no s390x support
-  skip: True  # [py<36 or (linux and s390x)]
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vvv
+  skip: True  # [py<36 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
   entry_points:
     - spacy = spacy.cli:setup_cli
 
 requirements:
   build:
     - {{ compiler('cxx') }}
+    - patch       # [not win]
+    - m2-patch    # [win]
   host:
+    - pip
     - python
-    - cymem 2.0.6
-    - cython 0.29.32
-    - murmurhash 1.0.7
     - numpy {{ numpy }}
-    - pathy 0.10.1
-    - pip
-    - preshed 3.0.6
     - setuptools
-    - thinc 8.1.10
     - wheel
+    - cython >=0.25,<3.0
+    - murmurhash >=0.28.0,<1.1.0
+    - cymem >=2.0.2,<2.1.0
+    - preshed >=3.0.2,<3.1.0
+    - thinc >=8.1.8,<8.3.0
   run:
     - python
     # Spacy libraries
     - spacy-legacy >=3.0.11,<3.1.0
     - murmurhash >=0.28.0,<1.1.0
     - cymem >=2.0.2,<2.1.0
     - preshed >=3.0.2,<3.1.0
-    - thinc >=8.1.8,<8.2.0
+    - thinc >=8.1.8,<8.3.0
     - wasabi >=0.9.1,<1.2.0
     - srsly >=2.4.3,<3.0.0
     - catalogue >=2.0.6,<2.1.0
     - spacy-loggers >=1.0.0,<2.0.0
-    # Exclude typer with unpinned click (>=0.3.2)
-    - typer >=0.3.0,<0.8.0
-    - pathy >=0.10.0
-    - smart_open >=5.2.1,<7.0.0
+    - typer >=0.3.2,<0.10.0
```

Comment:
According to https://inspector.pypi.io/project/spacy/3.7.2/packages/f7/a8/ce2bf530416327386d10f9ed577a600a5499987cb3368215b46c7b2f7221/spacy-3.7.2-cp39-cp39-win_amd64.whl/spacy-3.7.2.dist-info/METADATA#line.40, it should be `0.3.0,<0.10.0`.

Reply:
```suggestion
    - typer >=0.3.0,<0.10.0
```
thanks